core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/TestDriverComponent.scala

```diff
@@ -996,6 +996,7 @@ trait TestDriverComponent extends DriverComponent with DataAccess with DefaultIn
 
     override def save() = {
       DBIO.seq(
+        rawlsBillingProjectQuery.create(billingProject),
```

Comment:
there's no FK from workspace namespace to billing project name, so it flew under the radar until i actually needed it

Reply:
Yikes, wonder if we should  create a ticket to create a FK constraint between Billing Project name and Workspace namespace?  Seems like we should unless one of the gurus, aka Doug, tells us there was some legitimate reason why they didn't add this previously.  I wouldn't be surprised either way.

Reply:
I don't think it was intentionally done this way. My best guess is that it's because workspace "namespaces" existed before billing projects did. There was no need for that column to have a FK to anything because there was no other table to point to at the time. Once we implemented billing projects, we just never switched the column to point to a unique billing project ID. All of this was super early on in Rawls history and some of those oversights still live on :)